sentry_sdk/integrations/threading.py

```diff
@@ -21,9 +27,21 @@
 class ThreadingIntegration(Integration):
     identifier = "threading"
 
-    def __init__(self, propagate_hub=False):
-        # type: (bool) -> None
-        self.propagate_hub = propagate_hub
+    def __init__(self, propagate_hub=None, propagate_scope=True):
+        # type: (Optional[bool], bool) -> None
+        if propagate_hub is not None:
+            logger.warning(
+                "Deprecated: propagate_hub is deprecated. This will be removed in the future."
```

Comment:
Are we still using this anywhere in the codebase?

Reply:
No, but `propagate_hub` is public api: https://docs.sentry.io/platforms/python/integrations/default-integrations/#threading